server/server.js

```diff
@@ -183,9 +202,80 @@ app.post('/genotet/upload', upload.single('file'), function(req, res) {
   });
 });
 
-/**
- * GET request handler.
- */
+app.use(cookieParser());
+
+app.post('/genotet/user', function(req, res) {
+  log.serverLog('POST user');
+
+  var type = req.body.type;
+  MongoClient.connect(mongoUrl, function(err, db) {
+    assert.equal(null, err, err);
```

Comment:
Use logs to record error, not assert. 

Strictly speaking, verbally my last comment could be incorrect as it should be "Logs are preferred [to assertion](http://english.stackexchange.com/questions/1752/which-is-correct-prefer-x-to-y-or-prefer-x-over-y)" :)